src/applications/mhv-landing-page/selectors/index.js

```diff
@@ -2,24 +2,29 @@ import {
   isAuthenticatedWithSSOe,
   signInServiceEnabled,
 } from '~/platform/user/authentication/selectors';
-import { isLoggedIn, selectProfile } from '~/platform/user/selectors';
+import {
+  isLoggedIn,
+  selectProfile,
+  isLOA3,
+  isInMPI,
+} from '~/platform/user/selectors';
 import { selectDrupalStaticData } from '~/platform/site-wide/drupal-static-data/selectors';
 
 import { isLandingPageEnabled, personalizationEnabled } from './featureToggles';
 import { isLandingPageEnabledForUser } from './isLandingPageEnabledForUser';
 import { hasHealthData } from './hasHealthData';
+import { selectGreetingName } from './selectGreetingName';
 
 const selectVamcEhrData = state =>
   selectDrupalStaticData(state)?.vamcEhrData || {};
 
-const selectGreetingName = state =>
-  state?.user?.profile?.userFullName?.first || state?.user?.profile?.email;
-
 export {
   hasHealthData,
   isAuthenticatedWithSSOe,
+  isInMPI,
   isLandingPageEnabled,
   isLandingPageEnabledForUser,
+  isLOA3,
```

Comment:
Did you really add this, or is this a merge/PR issue showing this?

Reply:
I added it.